### DISCUSSION TITLE
Unable to open other repos gh pages

### DISCUSSION THREAD

**Discussion by ninjaasmoke** [community/community#23721] [Other Feature Feedback, Questions, & Ideas]
<p>I have created my personal gh page at <a href="https://ninjaasmoke.github.io" rel="noopener nofollow ugc">https://ninjaasmoke.github.io</a> my username is <code>ninjaasmoke</code>. I also have other gh pages for other repos (like <a href="https://ninjaasmoke.github.io/angular/module-2-assignment/" rel="noopener nofollow ugc">https://ninjaasmoke.github.io/angular/module-2-assignment/</a> ). But since I created my personal gh page, I’m unable to open the other repos’ pages! I thought it was because I was redirecting urls from my personal gh page, but that is not the case. Help!!!</p>

**Comment by otter-computer** [comment, marked as answer]
<p>Hi <a class="mention" href="/ninjaasmoke">@ninjaasmoke</a>, welcome to the GitHub Support Community!</p>
<p>This is likely because of the service worker you have enabled for your personal site <code>ninjaasmoke.github.io</code>. This service worker will be capturing all traffic to the <code>ninjaasmoke.github.io</code> domain—including visits to your other sites—and redirecting it to your personal page.</p>
<p>If you unregister the service worker for your personal page from your browser’s developer tools you should then be able to visit your other pages again. However, this will only work until the next time you visit your personal site where the service worker will be registered again.</p>
<p>To fix this permanently you’ll need to disable the service worker component of your personal site.</p>

**Comment by ninjaasmoke** [comment]
<p>Yes! That was the issue! Turns out I have a LOT to learn about service workers. Thanks</p>
